packages/components/src/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import classnames from "classnames";
+import styles from "./Spinner.css";
+
+interface SpinnerProps {
+  /**
+   * Specifies the size of the spinner
+   *
+   * @default "base"
+   */
+  readonly size?: "small" | "base";
+
+  /**
+   * Spinner becomes an inline element when true,
+   * otherwise it is a block element
+   */
+  readonly inline?: boolean;
+}
+
+export function Spinner({ size = "base", inline }: SpinnerProps) {
+  const spinnerStyles = classnames(styles.spinner, {
+    [styles.small]: size === "small",
+    [styles.inline]: inline,
+  });
+
+  return (
+    <div
+      className={spinnerStyles}
+      role="alert"
+      aria-busy={true}
+      aria-label="loading"
```

Comment:
@chris-at-jobber again thoughts on accessibility? based on #177, and the current roles in the custom spinner in JO.
I tried to test this with the mac voiceover but boy oh boy

Reply:
I've tested this with VoiceOver and it seems to work fairly well, one suggestion:

Currently it reads out "Empty alert: loading" because the screenreader expects content inside of an alert. Ideally it reads out "alert: loading" which I think we could achieve with something like the example [here](https://codepen.io/JasonAment/pen/abOJmdo) but, also, I think communicates the context decently as-is.

As a side note, if you need a hand testing with VoiceOver ping the accessibility guild channel! Lots of folks in there who can help you get started, it can be _a lot_ at first.